_includes/footer.html

```diff
@@ -21,7 +21,7 @@ <h5>Follow Us</h5>
                 <p class="network">
                     {% for network in site.data.footer.network %}
                         {% if network.linkedin %}
-                            <span> <a href="{{network.linkedin}}" target="_blank"> {% include icons/linkedin.svg %}</a></span>
+                            <span> <a href="https://www.linkedin.com/company/wcc-womencodingcommunity" target="_blank"> {% include icons/linkedin.svg %}</a></span>
```

Comment:
Ah, sorry that is missing is to create yml file  on this folder `_include` with name, for example footer.yml or configuration.yml with this information: 

```
network:
  - linkedin: https://www.linkedin.com/company/wcc-womencodingcommunity
  - github: https://github.com/WomenCodingCommunity
  - instagram: https://www.instagram.com/womencodingcommunity/
```   

Soon we are going to create the others. Sorry for the confusion.